CHANGELOG.md

```diff
@@ -1,11 +1,29 @@
 ### Unreleased
+
 - added option `max_depth` used to specify the maximum depth of level's schema that have to be rendered
```

Comment:
questa parte in che versione è uscita?
penso si sia rotto il changelog

Reply:
I don't know when. Compared to master I did not modify the changelog except by putting down the minor ones made in the original repo